kernel/src/main/java/org/kframework/kompile/DefinitionParsing.java

```diff
@@ -127,29 +120,8 @@ public DefinitionParsing(
         }
 
         def = resolveNonConfigBubbles(def, gen);
-        saveCachesAndReportParsingErrors();
```

Comment:
Place where caches were saved before - at the very end of `parseModules` invoked during runtime parsing. Similar for compile-time parsing.